src/styles/stars.scss

```diff
@@ -0,0 +1,19 @@
+.card__stars {
+  display: flex;
+}
+
+.stars_star {
+  width: 16px;
+  height: 16px;
+  flex-shrink: 0;
+
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+
+}
+
+.stars--4 :not(:last-child) {
+  margin-right: 4px;
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
Need to implement it
<img width="559" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/64750958/8f274bb9-c560-430f-8511-629d0764af12">